app/js/components/orderEntry/OrdersTable.jsx

```diff
@@ -9,21 +9,86 @@ import DrugOrderDetails from './DrugOrderDetails';
 import fetchOrders from '../../actions/fetchOrders';
 import { DRUG_ORDER } from './orderTypes';
 import { setSelectedOrder } from '../../actions/orderAction';
+import createLabOrder from '../../actions/createLabOrder';
+import { successToast, errorToast } from '../../utils/toast';
+import toggleUrgency from '../../actions/labOrders/toggleUrgency';
 
 export class OrdersTable extends PureComponent {
+  state = {
+    recentUrgency: '',
+    previousOrderId: '',
+  };
+
   componentDidMount() {
     this.props.dispatch(fetchOrders(null, this.props.patient.uuid));
   }
 
-  componentDidUpdate(prevProps, prevState) {
+  componentDidUpdate(prevProps) {
+    const { previousOrderId, recentUrgency } = this.state;
+    const {
+      status: { added, error },
+      errorMessage,
+      labOrderData,
+    } = this.props.createLabOrderReducer;
     if (this.props.patient.uuid !== prevProps.patient.uuid) {
       this.props.dispatch(fetchOrders(null, this.props.patient.uuid));
     }
+    if (added && labOrderData !== prevProps.createLabOrderReducer.labOrderData) {
```

Comment:
please what is does the variable name `added` represent ?

Reply:
well `added` doesn't really read so good, making it specific would make more sense

Reply:
did you look at the full code in this code block, `added` is a variable obtained from destructuring, It wasn't originally defined here.

and what is your idea of specific? please refer below to get an overview of what I am doing here.